R/driver.R

```diff
@@ -179,6 +213,13 @@ driver <- function(all_data = empty_data(),
     all_data <- add_data(csv_data, all_data)
   }
 
+  # Extract metadata from the input data; we'll add output metadata as we run
+  metadata_info <- list()
+  if(return_data_map_only) {
+    if(!quiet) cat("Extracting metadata from inputs...\n")
```

Comment:
Stuff like this should be done with `message`.  Then you can dispense with the `quiet` argument and just let the user use `suppressMessages` if they don't want to see the output (warning and error text will still come through even with messages suppressed.

Reply:
Solid point but since this is the current standard throughout, I'm going to leave it for now. Happy to revisit for v1.1.